services/api/database/migrations/20250619040511_add_file_to_auditlog_type.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function(knex) {
+    return knex.schema.
+        raw(`ALTER TABLE audit_log MODIFY resource_type ENUM('backup', 'bulkdeployment', 'deployment', 'deploytarget', 'deploytargetconfig', 'environment', 'group', 'notification', 'organization', 'project', 'sshkey', 'task', 'user', 'variable', 'workflow', 'file') NOT NULL;`).
+        raw(`ALTER TABLE audit_log MODIFY linked_resource_type ENUM('backup', 'bulkdeployment', 'deployment', 'deploytarget', 'deploytargetconfig', 'environment', 'group', 'notification', 'organization', 'project', 'sshkey', 'task', 'user', 'variable', 'workflow', 'file') NOT NULL;`);
```

Comment:
[These can be null](https://github.com/uselagoon/lagoon/blob/v2.25.0/services/api/database/migrations/20250218000000_audit_log.js#L14-L17) 
```suggestion
        raw(`ALTER TABLE audit_log MODIFY resource_type ENUM('backup', 'bulkdeployment', 'deployment', 'deploytarget', 'deploytargetconfig', 'environment', 'group', 'notification', 'organization', 'project', 'sshkey', 'task', 'user', 'variable', 'workflow', 'file');`).
        raw(`ALTER TABLE audit_log MODIFY linked_resource_type ENUM('backup', 'bulkdeployment', 'deployment', 'deploytarget', 'deploytargetconfig', 'environment', 'group', 'notification', 'organization', 'project', 'sshkey', 'task', 'user', 'variable', 'workflow', 'file');`);
```